dolphinscheduler-alert/dolphinscheduler-alert-plugins/pom.xml

```diff
@@ -39,6 +39,7 @@
         <module>dolphinscheduler-alert-pagerduty</module>
         <module>dolphinscheduler-alert-webexteams</module>
         <module>dolphinscheduler-alert-telegram</module>
+        <module>dolphinscheduler-alert-voice</module>
```

Comment:
I think it's better rename to `dolphinscheduler-alert-aliyunvoice` since there are other voice alerts available.

Reply:
completed